modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/endpoint/liferay/LiferayOAuthDataProvider.java

```diff
@@ -653,9 +653,9 @@ public void updateRememberDeviceContent(
 	@Activate
 	@SuppressWarnings("unchecked")
 	protected void activate(Map<String, Object> properties) {
-		_oAuth2AuthorizationServerConfiguration =
+		_defaultWebKeyProviderConfiguration =
 			ConfigurableUtil.createConfigurable(
-				OAuth2AuthorizationServerConfiguration.class, properties);
```

Comment:
If we reference DefaultWebKeyProviderConfiguration here, what happens when devs create their own impl of WebKeyProvider?